lib/config.js

```diff
@@ -1,5 +1,13 @@
-var os = require('os');
-var path = require('path');
+// External Modules
+const os = require('os');
+const path = require('path');
+
+// Internal Modules
+const pkg = require('../package.json');
+
+// Constants
+const PKG_SYSTEMS = { linux: '-linux' , win32: '-windows' , darwin: '-mac' };
```

Comment:
I personally think this makes the next line look messy; I think it's fine to just leave the logic like it was in index.js:28-36 before.

Reply:
I honestly loved this one since `LDUSERAGENT` `Environment Variable` can be used as well as a part of configuration.